Rename Binance Smart Chain to BNB Chain

RumeelHussainbnb opened the pull request:
the response received from the "https://chainid.network/chains.json" is used to enlist chain details. However due to missing api files (pages/api)on the github repo, one cant edit it. 

it is requested to update the name of **Binance Smart Chain** to **BNB Chain**.

Sleem3403 commented:
gh pr checkout 61

Djoudjasow11 commented:
Gddfg

Djoudjasow11 commented:
Thank

DevinAPieceOfSit reviewed (commented):
FR:@noxious15

NostalgicGareth commented:
pls

LToxiCs commented:
Que Bien ☣️

ghost commented:
hello

Debu976116 reviewed (approved):
approve

NostalgicGareth reviewed (approved):
Approve

NostalgicGareth reviewed (approved):
- [ ]

LToxiCs commented:
@LToxiCs

NostalgicGareth reviewed (approved):
- [ ]

NostalgicGareth reviewed (approved):
- [ ]

NostalgicGareth reviewed (approved):
- [ ]

RSivakov commented:
Upd

RSivakov commented:
Upd

G-822758 commented:
>

Ameralameri commented:
Approv

Ameralameri reviewed (approved):
Approv

Ameralameri reviewed (approved):
Approve

Ameralameri reviewed (approved):
Approve

Ameralameri reviewed (approved):
Ap

Chrisandjojo commented:
Trying my assets

zdeadex committed:
Add XDEFI support (#781)

realdealshaman committed:
add to translations

Commit:
Add Validatrium  Evmos endpoint

MusabShakeel576 committed:
Remove deprecated Fuse RPCs (#783)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

AutoStake1 committed:
Add AutoStake Evmos jsonrpc's (#789)

alkadeta committed:
Update extraRpcs.js (#795)

realdealshaman committed:
add autostake privacy policy

realdealshaman committed:
remove rpc

realdealshaman committed:
add new chains

realdealshaman committed:
add privacy policy

ayo-klaytn committed:
fix/outdated-klaytn-extraRPC (#800)

realdealshaman committed:
add tracking details

realdealshaman committed:
add stakin

Commit:
Add lokibuilder (#802)

* Add lokibuilder

* Move to the end of array

realdealshaman committed:
add cyphercore privacypolicy

jim380 committed:
Add Cypher Core EVMOS JSONRPC (#804)

realdealshaman committed:
add privacyStatement to rpc

JoGetBlock committed:
Update extraRpcs.js (#806)

Update with a relevant public RPC endpoint for GetBlock

realdealshaman committed:
add rpc from pr #801

alkadeta committed:
Update extraRpcs.js (#809)

realdealshaman committed:
fix syntax

alkadeta committed:
Update extraRpcs.js (#811)

wangdayong228 committed:
add chain_id for conflux in chainIds.json (#824)

DGSInfra committed:
Update extraRpcs.js (#823)

add 6363 and 363636

maxzonder committed:
Update extraRpcs.js add mzonder endpoints (#822)

KaliaNetwork committed:
Add Kalia json rpc (#821)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

3eph1r0th committed:
Add Allnodes RPC Endpoint for Bahamut (#820)

Northa committed:
add alphab evmos eth json rpc mainnet&testnet (#818)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

s0urledd committed:
Update extraRpcs.js (#816)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

silentnoname committed:
Update extraRpcs.js (#815)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

shawnonchain committed:
Update extraRpcs.js-add-1RPC-CORE-Support (#814)

qubelabsio committed:
Update extraRpcs.js (#810)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
fix syntax

imperator-co committed:
Add Imperator Evmos RPC (#819)

* Update extraRpcs.js

* Update extraRpcs.js

* Fix deleted RPC

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

VandaleCallender committed:
Update extraRpcs.js (#833)

Project Pi 

Mainnet - pulse-s .projectpi.xyz
Testnet - pulse-test.projectpi.xyz

realdealshaman committed:
add privacy policy

0xx92 committed:
Update extraRpcs.js (#830)

DGSInfra committed:
Update extraRpcs.js (#829)

change rpc 6363 and 363636 to https

0xx92 committed:
Update extraRpcs.js (#828)

realdealshaman committed:
fix syntax

chris-mercer committed:
Note Deprecated Chains 6, 62 (#838)

Note deprecated Ethereum Classic testnet chains to prevent replay attacks.

Chain 6 Kotti (PoA) and 62 Morden (PoW with old 2016 baggage).

Chain 63 Mordor is the preferred testnet for Ethereum Classic.

solidityx committed:
Added NetZ's New RPC (#836)

https://eu-rpc.mainnetz.io

VandaleCallender committed:
Update extraRpcs.js (#834)

updated testnet rpc url

bstr156 committed:
Rollux: Remove malfunctioning third-party RPC (#798)

RPC https://rollux.public-rpc.com is unstable and has been negatively impacting user experiences for weeks with no resolution. Will add again once issue is confirmed resolved and service is stable.

-- Syscoin Foundation, Rollux team

isabellewei committed:
Add Scroll mainnet extra RPC URLs (#842)

* add lava Evmos RPC endpoint as default (#735)

* Remove deprecated Fuse RPCs (#783)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

* add scroll mainnet extra RPCs

---------

Co-authored-by: MalteHerrmann <42640438+MalteHerrmann@users.noreply.github.com>
Co-authored-by: Musab Shakil <46605319+MusabShakeel576@users.noreply.github.com>
Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>
Co-authored-by: isabelle <isabelle@scroll.io>

aditya-manit committed:
Update extraRpcs.js (#841)

add KingSuper Evmos Testnet RPC

Commit:
feat(rpc): add defichain evm rpcs (#839)

* feat(rpc): add defichain evm rpcs

* feat(rpc): add more rpcs

HebeBlock committed:
Add ETC nodes and modify node information (#840)

* Add ETC nodes and modify node information

1.add rpc.etcinscribe.com
2.add etc.etcdesktop.com tracking

* Add ETC nodes

Add ETC nodes

chris-mercer committed:
Update: etc-network.info Limited Tracking setting (#837)

* Update: etc-network.info Limited Tracking setting

Update to reflect limited tracking for RPC endpoints.

Remove offline RPC endpoints

* Update extraRpcs.js

0xx92 committed:
Update extraRpcs.js (#850)

realdealshaman committed:
add beam

hybridfoundation committed:
Update Hybrid Chain 2468 Mainnet (#852)

* Update Hybrid Chain 2468 Mainnet

Update Hybrid Chain 2468 Mainnet

* Update chainIds.json

remove chain not yet added to defillama tvl charts

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

0xx92 committed:
Update extraRpcs.js (#805)

denalimarsh committed:
add ankr rpc

0xx92 committed:
Update extraRpcs.js (#855)

realdealshaman committed:
push update

0xngmi committed:
remove deprecated chains

0xngmi committed:
Merge pull request #857 from denalimarsh/main

Add ANKR RPC to CoreDAO chain

0xngmi committed:
Merge pull request #858 from DefiLlama/deprecated

remove deprecated chains

Tokenview committed:
Add tokenview RPC service (#861)

Co-authored-by: liukai <kai.liu@elegine.ai>

diega committed:
Add Rivet's privacy policy (#859)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
update order

sunil301295 committed:
Update extraRpcs.js (#862)

* Update extraRpcs.js

* Update extraRpcs.js

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
update logo

nanthanwa committed:
Add RPCs for JIBCHAIN (#863)

* add jibchain rpc

* move to end of array as suggestion

* Scope only inan.in.th

* remove privacy policy for inan.in.th

realdealshaman committed:
add redecoin

0xx92 committed:
Update extraRpcs.js (#879)

BelialNarcissus committed:
Update extraRpcs.js (#872)

* Update extraRpcs.js

Add nodeconnect rpc

* Update extraRpcs.js

tengokuvision committed:
add: Humans RPCs by ITRocket (#877)

* add: Humans RPCs to extraRpcs.js by ITRocket

* Update: tracking policy in extraRpcs.js for ITRocket

* use quote from privacy policy

update to quote from privacy policy

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
fix syntax

realdealshaman committed:
add rpc

alkadeta committed:
Update extraRpcs.js (#884)

alkadeta committed:
Remove duplicate entries from Evmos rpc list v2 (#886)

* Update extraRpcs.js

* Update extraRpcs.js

fanbsb committed:
update fusion network rpc (#893)

Danielvul committed:
Added OnFinality Eth Sepolia & Polygon Mumbai (#892)

boyuan-chen committed:
Add boba RPC (#890)

realdealshaman committed:
add chainid 131 rpcs

shawnonchain committed:
Update extraRpcs.js_add_1RPC_Telos_Kroma_Mode (#897)

Added 1RPC endpoints for networks Telos, Kroma and Mode

molecula451 committed:
chore: remove dead nodes (#895)

realdealshaman committed:
fix syntax

realdealshaman committed:
update arb list

icodezjb committed:
Update chainid=1501 rpc urls (#901)

use new urls

bishalghimire77 committed:
Update extraRpcs.js (#898)

* Update extraRpcs.js

added siriusnet rpc

* Update extraRpcs.js

fix alignment

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

MohitXDC committed:
Update extraRpcs.js (#904)

Deletion request of an unsupported RPC

0xx92 committed:
Update extraRpcs.js (#902)

AIDidMyHomework committed:
Update extraRpcs.js (#899)

* Update extraRpcs.js

Added Decubate's Linea node

* Update extraRpcs.js

set tracking to yes until we've updated our policy page

* Update extraRpcs.js

updated the privacy policy

fulldecent committed:
Rename to ChainList, per #593

aomwara committed:
update: add rpc (#907)

0xngmi committed:
Merge pull request #906 from fulldecent/patch-1

Rename to ChainList, per #593

AIDidMyHomework committed:
Update extraRpcs.js (#905)

Updated privacy statement and published privacy policy

bfloresio committed:
Update kava rpcs (#910)

update kava rpcs to include ankr.

0xGery committed:
Update extraRpcs.js (#909)

* Update extraRpcs.js

Link the service provider's website: https://nocturnode.tech
Provide a link to your privacy policy: N/A, open source, free to use

* Update extraRpcs.js

* Update extraRpcs.js

* Update extraRpcs.js

delete tracking details

* Update extraRpcs.js

fix entry

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

0xGery committed:
Update extraRpcs.js (adding Privacy policy of Nocturnode.tech) (#913)

* Update extraRpcs.js

* Update extraRpcs.js

add privacy statement

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

bfloresio committed:
add ankr endpoint kava (#912)

add ankr endpoint kava

realdealshaman committed:
add polysplit privacy statement

realdealshaman committed:
add polysplit

realdealshaman committed:
fix

realdealshaman committed:
fix tracking

realdealshaman committed:
rpc issue

guo committed:
Update IoTeX RPC (#911)

* Update IoTeX RPC

* Update extraRpcs.js

realdealshaman committed:
add reference url

0xx92 committed:
Update extraRpcs.js (#917)

0xx92 committed:
Update extraRpcs.js (#919)

ryujimin220 committed:
Update extraRpcs.js with new All That Node endpoints for ZetaChain (#920)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

0xx92 committed:
Update extraRpcs.js (#918)

realdealshaman committed:
update tracking info for allthatnode

tengokuvision committed:
Update: ITRocket endpoints and Privacy Policy (#923)

* Update: ITRocket endpoints and privacy policy

* fix: typo

AM-979 committed:
Update extraRpcs.js (#924)

Add rpc

mnedelchev-vn committed:
deprecate https://proxy.devnet.neonlabs.org/solana rpc (#926)

* deprecate https://proxy.devnet.neonlabs.org/solana rpc

* Update extraRpcs.js

update

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

kotsmile committed:
fix polysplit (#925)

bford21 committed:
Add Ansybl Canto RPC (#933)

* Add Ansybl Canto RPC

* Update extraRpcs.js

add new rpc to end of the list

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

pertsevalexey committed:
Update extraRpcs.js (#932)

0xx92 committed:
Update extraRpcs.js (#930)

Parastoo1234 commented:
Hi

alxnode committed:
Update extraRpcs.js (#935)

add dymension

nodestake committed:
Add Dymension Jsonrpc by NodeStake (#934)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

Andrew-Pohl committed:
Add dymension jsonrpc by Liquify (#937)

Nguyendinhtuan17 commented:
> the response received from the "https://chainid.network/chains.json" is used to enlist chain details. However due to missing api files (pages/api)on the github repo, one cant edit it.
> 
> it is requested to update the name of **Binance Smart Chain** to **BNB Chain**.

pertsevalexey committed:
Update extraRpcs.js (#939)

BitOfUniverse committed:
Add zeta.rpcgrid.com RPCs (+WS) for ZetaChain (#931)

* Add zeta.rpcgrid.com RPCs (+WS) for ZetaChain

* Use SSL for zeta.rpcgrid.com
use portless hosts
add privacy policy

kynraze committed:
Update extraRpcs.js (#941)

ivanshukhov committed:
add extra rpc (#946)

0xx92 committed:
Update extraRpcs.js (#949)

mkdirken committed:
Add Areon Chain Rpc (#948)

OfTheWolf committed:
Lava eth mainnet  (#952)

* Update extraRpcs.js

* Update extraRpcs.js

update tracking to match the privacystatement

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

isabellewei committed:
Add DRPC and POKT endpoints to Scroll (#951)

* add lava Evmos RPC endpoint as default (#735)

* Remove deprecated Fuse RPCs (#783)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

* add scroll mainnet extra RPCs

* add drpc and pokt endpoints to scroll

---------

Co-authored-by: MalteHerrmann <42640438+MalteHerrmann@users.noreply.github.com>
Co-authored-by: Musab Shakil <46605319+MusabShakeel576@users.noreply.github.com>
Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>
Co-authored-by: isabelle <isabelle@scroll.io>

3eph1r0th committed:
Update PublicNode (#954)

chenjia404 committed:
add rpc (#953)

JoGetBlock committed:
Update extraRpcs.js (#960)

Add GetBlock bcs endpoint to the list

robbieshadow committed:
Update extraRpcs.js (#959)

add rpc for eth mainnet 
evmos mainnet 
evmos testnet

realdealshaman committed:
add trackingdetails

realdealshaman committed:
add privacy statement for q org

KyrylR committed:
Add Q Mainnet RPC (#958)

* Added Q Mainnet RPC

* Update chainIds.json

remove chain not yet listed on DefiLlama

* Update extraRpcs.js

add privacy info

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

0xx92 committed:
Update extraRpcs.js (#957)

jessicadaugherty committed:
Update extraRpcs.js - Add Stateless provider on ETH, Polygon, OP and ARB (#956)

* Update extraRpcs.js

- Added privacyStatement for new RPC provider (Stateless)
- Added Stateless RPC endpoint details for ETH, Polygon, and OP

* Update extraRpcs.js - Add Stateless provider

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
fix syntax

noirzz committed:
Update extraRpcs.js (#962)

0xx92 committed:
Update extraRpcs.js (#967)

0xngmi committed:
remove scam

0xngmi committed:
add ankr blast

realdealshaman committed:
add blast rpcs

wengzilla committed:
feat: Upgrading to new HypeLab SDK. Preventing ad refresh on keyDown.

Commit:
Add blast.gasswap.org

0xngmi committed:
Merge pull request #971 from zerotwodao/main

Add blast.gasswap.org

Commit:
update blast.gasswap.org ws endpoint

0xngmi committed:
Merge pull request #972 from zerotwodao/main

update blast.gasswap.org ws endpoint

fernando-syslabs committed:
Remove wrong RPCs from Syscoin NEVM testnet list. (#973)

Co-authored-by: Fernando Paredes Garcia <fernando@develcuy.com>

blockjoe committed:
make stateless URLs human readable (#980)

0xx92 committed:
Update extraRpcs.js (#978)

realdealshaman committed:
add new publicnode rpcs

DRIgnazGortngschirl committed:
Update extraRpcs.js (#938)

Add new RPC node & Update privacy statement and tracking info

JohnRising committed:
Update extraRpcs.js (#944)

* Update extraRpcs.js

Added stackup public RPCs

* Update extraRpcs.js

use quote from privacy policy

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
fix syntax

AmardeepSambaru committed:
Update extraRpcs.js (#979)

* Update extraRpcs.js

Added the Etherlink Ghostnet RPC

* use quote from privacy policy

use quote from privacy policy

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

wengzilla committed:
fix: Undoing single quote lints. Removing other changes from yarn.lock. Removing package-lock.json.

charlie-eth committed:
Merge pull request #969 from wengzilla/hypelab-upgrade

chore + fix: Upgrading HypeLab SDK version and fixing re-rendering issue.

JoGetBlock committed:
Update extraRpcs.js (#985)

update Getblock's endpoint

fernando-syslabs committed:
Update Allnodes RPC URLs for Syscoin Testnet and Mainnet (#988)

* Remove wrong RPCs from Syscoin NEVM testnet list.

* Update Allnodes RPC URLs for Syscoin Testnet and Mainnet.

---------

Co-authored-by: Fernando Paredes Garcia <fernando@develcuy.com>

JoGetBlock committed:
Update extraRpcs.js (#986)

Update the BSC URL

eosiodetroit-adam committed:
Add Detroit Ledger Tech Telos EVM RPC Endpoint (#989)

* Add Detroit Ledger Tech Telos EVM RPC Endpoint

* Update extraRpcs.js

Added /evm to endpoint.

itsmnthn committed:
feat: add hubblenet (#977)

realdealshaman committed:
add hypr network

jincan39 committed:
Add Manta Additional RPC URL (#993)

0xx92 committed:
Update extraRpcs.js (#999)

blockchaingirl1407 committed:
Adding Tatum RPC - Aurora Mainnet (#997)

tornadocontrib committed:
Delist scam rpc ( tornadoeth.cash )

0xngmi committed:
Merge pull request #1000 from tornadocontrib/main

Delist scam rpc ( tornadoeth.cash )

realdealshaman committed:
add new chains

realdealshaman committed:
add new additionalChainRegistry folder

sheenhx committed:
Update extraRpcs.js (#1006)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

jbuice committed:
Update extraRpcs.js (#1002)

The Telos Core Developers released our new v1.5 EVM software today. These updates remove defunct RPCs and add new ones.

thedarkjester committed:
add Linea RPC urls (#995)

Danielvul committed:
Added SubQuery RPC for Blast & Ethereum (#987)

* Added SubQuery RPC for Blast & Ethereum

* Update extraRpcs.js

remove tracking with no privacy policy

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
fix syntax

realdealshaman committed:
update info

blockchaingirl1407 committed:
Adding Tatum BSC RPC node (#1014)

blockchaingirl1407 committed:
Addidng Tatum BSC RPC Node (#1013)

blockchaingirl1407 committed:
Adding Tatum Avalanche C Chain Node (#1012)

blockchaingirl1407 committed:
Adding Tatum BCH RPC node (#1011)

blockchaingirl1407 committed:
Adding Tatum RPC Arb Nova node (#1009)

blockchaingirl1407 committed:
Adding Tatum RPC Ethereum node (#1008)

realdealshaman committed:
fix syntax

0xngmi committed:
overwrite chains

realdealshaman committed:
add ronin

0xngmi committed:
fix imports

0xngmi committed:
fix imports

0xx92 committed:
Update extraRpcs.js (#1037)

Danielvul committed:
SubQuery Endpoint changes (#1036)

Updated Subquery's Ethereum & Base endpoints

Added Subquery Polygon, Arbitrum, and Eth Sepolia

JelliedOwl committed:
Update Jelliedowl RPCs (#1033)

I've rearranged the addresses for my RPCs - this corrects them to the new values.

realdealshaman committed:
add zeta chain

blockchaingirl1407 committed:
Adding Tatum RPC Celo Mainnet - removeing RPC Bitcoin Mainnet RPC bcs of wrong segmentation (#1015)

blockchaingirl1407 committed:
Adding Tatum RPC Doge (#1016)

blockchaingirl1407 committed:
Addint Tatum RPC node - Fantom (#1017)

blockchaingirl1407 committed:
Adding Tatum RPC node - Haqq (#1018)

blockchaingirl1407 committed:
Adding Tatum RPC node Gnosis (#1019)

blockchaingirl1407 committed:
Addint Tatum RPC node - Klaytn (#1020)

blockchaingirl1407 committed:
Adding Tatum RPC Polygon node (#1021)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

blockchaingirl1407 committed:
Adding Tatum RPC node - OP (#1022)

blockchaingirl1407 committed:
Adding Tatum RPC Node - Palm (#1023)

blockchaingirl1407 committed:
Adding Tatum RPC Node - XDC (#1024)

blockchaingirl1407 committed:
Adding Tatum RPC node - Base (#1025)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

art-eth committed:
Adding 2 RPC nodes - Cronos (#1028)

* Adding 2 RPC nodes - Cronos

* Update extraRpcs.js

BitrockChain committed:
Add new RPC | Bitrock chain (#1027)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

Ebrinix committed:
Add Egochain RPC (#1005)

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

ersanyakit committed:
chiliz chain integration (#961)

chiliz chain integration

realdealshaman committed:
update

dumitrucatalin committed:
Add latest OMNIA RPCs (#1039)

* Add Polygon zkEVM, ZKFair, ARB Sepolia, ZKSync Era, OKX, and Lightlink

* Add ETH Holesky and OP Sepolia

NodifiAI committed:
Update extraRpcs.js (#1026)

git commit NodifiAI free RPC endpoint

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
fix syntax

AnilChinchawale committed:
Update XDC Network RPCs (#1043)

- Added ETH Compatible WS
- Remove prefix RPCs

0xx92 committed:
Update extraRpcs.js (#1042)

boyuan-chen committed:
Add boba Sepolia network (#1041)

* Add boba Sepolia network

* Move it to the end of the array

3eph1r0th committed:
Add Polygon Bor Amoy (#1051)

masterbd committed:
fix: remove broken blastapi.io url (#1050)

0xx92 committed:
Update extraRpcs.js (#1047)

Macaromswap committed:
Update extraRpcs.js (#1062)

add bitlayer

0xx92 committed:
Update extraRpcs.js (#1061)

0xx92 committed:
Update extraRpcs.js (#1060)

0xx92 committed:
Update extraRpcs.js (#1059)

georgefuru committed:
Updated extraRpcs.js for Oasys. (#1055)

0xx92 committed:
Update extraRpcs.js (#1053)

realdealshaman committed:
add chain to list

irzhywau committed:
Update extraRpcs.js, added new RPC for elastos (20) (#1068)

intosKai committed:
Update extraRpcs.js (#1067)

JoGetBlock committed:
Update extraRpcs.js (#1066)

Add GetBlock ZkSync public endpoint to the list

art-eth committed:
Update extraRpcs.js (#1065)

ericHwangMetabora committed:
add borachain rpc (#1064)

Co-authored-by: ERIC-HWANG <eric@ERIC-HWANG.local>
Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

realdealshaman committed:
add rpc

realdealshaman committed:
add rpc

coolestowl committed:
Update extraRpcs.js (#1072)

* Update extraRpcs.js

add rockx node for merlin chain

* Update extraRpcs.js

fix privacy police

* Update extraRpcs.js

update tracking lable

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

0xx92 committed:
Update extraRpcs.js (#1076)

realdealshaman committed:
add rpc

jonathanlmatos committed:
Added Pollum services Rpc for syscoin and rollux, mainnet and testnet (#1077)

UmeyrFreezer committed:
Create tr.json (#1082)

JordyDutch committed:
Add SigmaCore RPC (#1080)

Stonepapa committed:
Update extraRpcs.js (#1083)

Add BSquared RPC endpoint

Quevedin77 committed:
Update extraRpcs.js - Taiko.Tools (#1078)

mskutlu committed:
Update extraRpcs.js lava eth (#1085)

coolestowl committed:
Update extraRpcs.js (#1088)

remove rockx endpoint

0xGraffiti committed:
Add Graffiti Full Node to Chainlist (#1087)

* Update extraRpcs.js

* Update extraRpcs.js

added excerpt from privacy policy and update to limited since IP is help for 7 days

---------

Co-authored-by: Real Shaman <85087525+realdealshaman@users.noreply.github.com>

lthms committed:
Update extraRpcs.js (#1071)

Add Etherlink mainnet. Currently, the network is not launched yet.

Danielvul committed:
Update extraRpcs.js (#1091)

Removed outdated SubQuery endpoints

0xx92 committed:
Update extraRpcs.js (#1090)

NOWNodes committed:
Update extraRpcs.js + NOWNodes BSC&ETH (#1093)

* Update extraRpcs.js + NOWNodes BSC&ETH

* Update extraRpcs.js + NOWNodes BSC&ETH

nowchain committed:
Add NOW RPC for NOW Chain (#1092)

RogerLamTd committed:
add ankr endpoint (#1096)

Joaquinpilotzi commented:
0x3aA1be1104720EA5dAe6c5412B34751E375D1004

Jay3445 commented:
0x57d31ef2cb561e9e0a0267c5051fa59eef8600e2

On Sat, May 18, 2024 at 16:29 Joaquin ***@***.***> wrote:

> 0x3aA1be1104720EA5dAe6c5412B34751E375D1004
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/antonnell/networklist-org/pull/61#issuecomment-2118692883>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARVYZCCKQAPBMUDWIHXE3RTZC4GNJAVCNFSM5QIXHCHKU5DIOJSWCZC7NNSXTN2JONZXKZKDN5WW2ZLOOQ5TEMJRHA3DSMRYHAZQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

iamcryptofennec committed:
Add PLYR TAU Testnet (#1095)

realdealshaman committed:
add rpcs

codev911 committed:
add more amoy rpc (#1100)

michaelotis committed:
Add L3x Protocol (#1105)

realdealshaman committed:
remove chain not yet listed at defillama

maxsiz committed:
Envelop RPC for Blast Mainnet (#1107)

Carlo0704 reviewed (approved):
👍

Joaquinpilotzi reviewed (commented):
Hoyefccu

Jay3445 commented:
how to received

On Sun, Jun 16, 2024 at 23:39 Joaquin ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> Hoyefccu
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/antonnell/networklist-org/pull/61#pullrequestreview-2121499944>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARVYZCBA3X3N35BEE4FAJ6DZHWWTHAVCNFSM5QIXHCHKU5DIOJSWCZC7NNSXTPCQOVWGYUTFOF2WK43UKJSXM2LFO45TEMJSGE2DSOJZGQ2A>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
